### ISSUE TITLE
TTT3 Medal/Ribbon camera preset

### ISSUE THREAD

**Issue by dmordred** [emperorshammer/issues-incoming-queue#201]
<salute>
Briefing:
As per the discussion on Nov 1st at Pliff's Asteroid Bar, the subject of being able to print the ribbons came about, and due to the hard work of LC Gytheran a solution was found on spot using the already available TTT3 engine.

The idea:
Add a camera preset that closes up on the ribbon/medals as such:
Position: x 70 y-350 z 175   Look at:  x37  y0  z115

It works for most cases as shown bellow:

![Mordred_Dressribbon](https://github.com/emperorshammer/issues-incoming-queue/assets/8151279/46804464-4fe8-4df3-ba51-81a628525ca1)
![camera referance](https://github.com/emperorshammer/issues-incoming-queue/assets/8151279/1db115ac-926c-41d7-a65f-45efbd2042a8)

**Comment by SmTaUcM** [comment]
I caught this thread this morning, I found it quite interesting and a neat idea. It would also be pretty easy to remove the uniform, buttons rank etc and provide a transparent background so you just had the ribbons and medals ready for the user to save.

**Comment by SmTaUcM** [comment]
And done! :

![Medals Only](https://github.com/emperorshammer/issues-incoming-queue/assets/32555971/1c8ce2e9-c935-45cf-b82e-902dfa636d5f)
![Medals Only Render](https://github.com/emperorshammer/issues-incoming-queue/assets/32555971/1ce66ed6-c8da-404c-9a26-74f7ec8f8d8a)

Pending the IOs review and approval for the next release.

**Comment by SmTaUcM** [comment]
Possibly overthinking things as I do, but i figured that we could go one better with this concept. So I've managed to remove the rotation out of all of the medals and ribbons so that they no longer conform to the uniform's bust and are now perfectly straight. The camera and look positions have been fine tuned so they no longer look at the "sweet spot" we found, but rather dead on with respect to the medals and ribbons.

![Medals Only](https://github.com/emperorshammer/issues-incoming-queue/assets/32555971/546ed242-568b-404c-a0e7-a034d0861203)

![Medals Only Render](https://github.com/emperorshammer/issues-incoming-queue/assets/32555971/9924f595-f94f-4f68-bf57-414fe5441095)
